src/main/ml-modules/root/runDuringDeployment/dataConstants/dataConstantsGenerator.mjs

```diff
@@ -1,349 +0,0 @@
-import { SPARQL_PREFIXES } from '../../lib/appConstants.mjs';

```

Comment:
Goodbye, Data Constants Generator!  You served us well.  Best of luck on the reincarnated bytes front!